doc/.howto-template.md

```diff
@@ -58,7 +58,7 @@
 <br>
 
 > [!Tip]
-> For a step-by-step guide to installing the prerequisites for your preferred IDE and environment, consult the [Get Started guide](../get-started.md).
+> For a step-by-step guide to installing the prerequisites for your preferred IDE and environment, consult the [Get Started guide](../get-started.md). <!-- Update the link if needed and remove this comment -->
```

Comment:
This serves as a reminder for when someone uses this template to not forget to update the link if required. Let me know if you want me to revert this particular change.